contracts/KSFactory.sol

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+
+import "./interfaces/IKSFactory.sol";
+import "./KSPool.sol";
+
+contract KSFactory is IKSFactory {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    uint256 internal constant BPS = 10000;
+
+    address private feeTo;
+    uint16 private governmentFeeBps;
+    address public override feeToSetter;
+
+    /// @dev fee to set for pools
+    mapping(uint16 => bool) private feeOptions;
```

Comment:
how to check if a fee option has been enabled if it is private?

Reply:
It's is public now.